File: src/test/java/edu/kit/iti/formal/stvs/view/common/ErrorMessageDialogDemo.java
package edu.kit.iti.formal.stvs.view.common;

import edu.kit.iti.formal.stvs.Demo;
import javafx.application.Application;
import javafx.stage.Stage;
import org.junit.experimental.categories.Category;

/**
 * Created by csicar on 01.02.17.
 */
@Category(Demo.class)
public class ErrorMessageDialogDemo extends Application {

  @Override
  public void start(Stage stage) throws Exception {
    AlertFactory.createAlert(new Exception("Test")).showAndWait();
  }
}

File: stvs/src/test/kotlin/edu/kit/iti/formal/stvs/view/common/ErrorMessageDialogDemo.kt
package edu.kit.iti.formal.stvs.view.common

import edu.kit.iti.formal.stvs.view.common.AlertFactory.createAlert
import javafx.application.Application
import javafx.stage.Stage
import org.junit.jupiter.api.Tag

/**
 * Created by csicar on 01.02.17.
 */
@Tag("demo")
class ErrorMessageDialogDemo : Application() {
    @Throws(Exception::class)
    override fun start(stage: Stage) {
        createAlert(Exception("Test")).showAndWait()
    }
}